This patch is for speech-tools for Debian GNU/Linux,
and forces the use of the actual template rather than that which
is inherits from when defining Iter typedef.

Index: b/include/EST_TIterator.h
===================================================================
--- a/include/EST_TIterator.h
+++ b/include/EST_TIterator.h
@@ -228,7 +228,7 @@
 
 public:
 
-  typedef EST_TIterator<Container, IPointer, Entry> Iter;
+  typedef EST_TRwIterator<Container, IPointer, Entry> Iter;
 
   /// Create an iterator not associated with any specific container.
   EST_TRwIterator() {this->cont=NULL;}
@@ -278,7 +278,7 @@
 {
 public:
 
-  typedef EST_TIterator<Container, IPointer, Entry> Iter;
+  typedef EST_TRwStructIterator<Container, IPointer, Entry> Iter;
 
   /// Create an iterator not associated with any specific container.
   EST_TRwStructIterator() {this->cont=NULL;}
